python/phylanx/ast/physl.py

```diff
@@ -383,12 +383,17 @@ def _compile_or_load(self):
                 check_return(self.ir)
 
                 self.__src__ = self._generate_physl(self.ir)
+                if self.doc_src is not None:
+                    self.__src__ = self.doc_src
```

Comment:
I think that if the doc-string should be used as the PhySL there is no reason to go run the `_generate_physl()` function here as well.